src/test/java/api/InstancesApiExamples.java

```diff
@@ -187,6 +189,76 @@ public void canCreateAnInstanceWithAnID()
+      .put("instanceTypeId", ApiTestSuite.getTextInstanceType());
+    JsonObject request = new JsonObject();
+    request.put("instances", new JsonArray()
+      .add(angryPlanetInstance)
+      .add(treasureIslandInstance));
+    request.put("totalRecords", 2);
+
+    // Post collection of instances
+    CompletableFuture<Response> postCompleted = new CompletableFuture<>();
+    okapiClient.post(ApiRoot.instancesCollection(), request, ResponseHandler.any(postCompleted));
+    Response postResponse = postCompleted.get(5, TimeUnit.SECONDS);
+    assertThat(postResponse.getStatusCode(), is(HttpResponseStatus.CREATED.code()));
+    assertEquals(postResponse.getJson().getJsonArray("instances").size(), 2);
+    assertEquals(postResponse.getJson().getJsonArray("errorMessages").size(), 0);
+    assertEquals(postResponse.getJson().getInteger("totalRecords"), Integer.valueOf(2));
+
+    // Get and assert angryPlanetInstance
+    CompletableFuture<Response> getAngryPlanetInstanceCompleted = new CompletableFuture<>();
+    okapiClient.get(String.format("%s/%s", ApiRoot.instances(), angryPlanetInstanceId), ResponseHandler.json(getAngryPlanetInstanceCompleted));
+    Response getAngryPlanetInstanceResponse = getAngryPlanetInstanceCompleted.get(5, TimeUnit.SECONDS);
+    assertThat(getAngryPlanetInstanceResponse.getStatusCode(), is(HttpResponseStatus.OK.code()));
+    JsonObject createdAngryPlanetInstance = getAngryPlanetInstanceResponse.getJson();
+    assertEquals(createdAngryPlanetInstance.getString("id"), angryPlanetInstanceId);
+    assertThat(createdAngryPlanetInstance.getString("title"), is("Long Way to a Small Angry Planet"));
+    assertThat(createdAngryPlanetInstance.getString("source"), is("Local"));
+    assertThat(createdAngryPlanetInstance.getString("instanceTypeId"), is(ApiTestSuite.getTextInstanceType()));
+
+    // Get and assert treasureIslandInstance
+    CompletableFuture<Response> getTreasureIslandInstanceCompleted = new CompletableFuture<>();
+    okapiClient.get(String.format("%s/%s", ApiRoot.instances(), treasureIslandInstanceId), ResponseHandler.json(getTreasureIslandInstanceCompleted));
+    Response getTreasureIslandInstanceResponse = getTreasureIslandInstanceCompleted.get(5, TimeUnit.SECONDS);
+    assertThat(getTreasureIslandInstanceResponse.getStatusCode(), is(HttpResponseStatus.OK.code()));
+    JsonObject createdTreasureIslandInstance = getTreasureIslandInstanceResponse.getJson();
+    assertEquals(createdTreasureIslandInstance.getString("id"), treasureIslandInstanceId);
+    assertThat(createdTreasureIslandInstance.getString("title"), is("Treasure Island"));
+    assertThat(createdTreasureIslandInstance.getString("source"), is("MARC"));
+    assertThat(createdTreasureIslandInstance.getString("instanceTypeId"), is(ApiTestSuite.getTextInstanceType()));
+  }
+
+  @Test
+  public void shouldReturnBadRequestIfPostedInstanceWithoutTitle() throws MalformedURLException, InterruptedException, ExecutionException, TimeoutException {
```

Comment:
What happens if two instances are sent together, one has a title and the other does not?

It could be useful to add a test to demonstrate that behaviour